.travis.yml

```diff
@@ -19,6 +19,7 @@ matrix:
     - env: IMAGE=dist-arm-linux DEPLOY=1
     - env: IMAGE=dist-armv7-aarch64-linux DEPLOY=1
     - env: IMAGE=dist-freebsd DEPLOY=1
+    - env: IMAGE=dist-fuchsia DEPLOY=1
```

Comment:
Right now we've actually got another similar container, `cross` above, which just builds a couple of targets that we release. Could you fold this container into that one and just add the targets there?

That container doesn't take too long to build so we should definitely be able to get a full build in under three hours, and that way we don't have to burn an extra slot on a new travis builder :)